src/css/custom.css

```
@@ -271,4 +273,11 @@ blockquote p::after,
 .markdown blockquote p::after {
   content: none !important;
   display: none !important;
+}
+
+p{
+  line-height: 1.5;
+}
```

1.6 should probably be the minimum here
```suggestion
p {
  line-height: 1.6;
}
```